### ISSUE TITLE
Show date of dataset publication on top

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#3393]
**Show date of dataset publication on top**

The publication date of the data is argueably more of interest then the date the metadata was last changed. It should be shown at the top so people have an idea how old the actual data is. Even if it's not always existing.

Publication date March 3, 2021

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/930aa253a84aee9732aa0f23ea964bc4:d212e45a70828c5d3f8f7fe0a5d9d5d98f3a9412242d72351d3dff63f90e1431f902565d0b9235a6a756e3c47e96a0be134dd8a89eb321ad0f2e130fd2520e01)
System: Safari 14.0.2 / Mac OS X 10.15.6
Referer: https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c
Window size: width 1759 - height 975
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-04-26T15:35:10.411Z',to:'2021-04-26T15:41:10.411Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-04-26T15:35:10.411Z',to:'2021-04-26T15:41:10.411Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by MortenHofft** [comment]
Seems reasonable. It is now added to the top of the page as well. It could be supplemented with a FAQ. I for one do not know what it refers to. 

* modified date: it controlled by us and refers to the metadata change
* created date: it controlled by us and refers to the creation of the dataset in the registry
* published date: It is my impression that it is updated at the publishers discretion. Which might be worth pointing out?

Perhaps a general FAQ to explain what those dates refer to? The API would also be a good place to document what fields mean, but documenting response values isn't something that we do in our current documentation.

This is related to https://github.com/gbif/portal-feedback/issues/3374

**Comment by njdowdy** [comment]
This is a welcomed addition. I would agree with @MortenHofft that some explanation of the terms should be given, as it is not clear to me precisely what the difference between 'registration' and 'publication' dates is:

> 
Registration date
    March 2, 2011
Metadata last modified
    April 26, 2021
Publication date
    March 3, 2021 
    
Also, is a registration date of March 2, 2011 correct? Or should that be 2021? Again, my poor understanding of what the term 'registration' means in this context makes it difficult to know.

**Comment by vijaybarve** [comment]
Is this correct understanding?

- Registration date March 2, 2011 is when the dataset was first created in the GBIF system
- Publication date March 3, 2021 is last update of the dataset

Metadata last modified April 26, 2021 is clear.

**Comment by MortenHofft** [comment]
@njdowdy 
> Also, is a registration date of March 2, 2011 correct? Or should that be 2021? Again, my poor understanding of what the term 'registration' means in this context makes it difficult to know.

Created/Registered means that this dataset was published-to/registered-with GBIF in 2011. In other words - GBIF have had this dataset in its registry of datasets since 2011. The dataset might have existed since 2009, but it was first published to GBIF in 2011. It might also have been updated many times since. This date is under GBIF control

@vijaybarve
>Publication date March 3, 2021 is last update of the dataset

It is a reflection of what the publisher writes. Not all publishers use it. And there is no guarantee that it is updated. But the EML standard says that it should be updated every time data or metadata changes. This date is not under GBIF control

**Comment by njdowdy** [comment]
@MortenHofft thanks for the clarification. Something like that should be found on the dataset page so that people who are not lurking on github can understand the definitions better too. 

I would suggest though, that GBIF simply state what each field is. For example, even your definition of "registration" includes the phrase "first **_published_** to GBIF in 2011", potentially confusing it with "publication" date. You can see how these sound very similar. 

I would suggest the following:

First Published on GBIF: # Registration date
March 2, 2011
Metadata last modified
April 26, 2021
Last Publisher Update: # publication date
March 3, 2021

These phrases are longer, but a more precise definition is built into the label.

**Comment by mdoering** [comment]
> > Publication date March 3, 2021 is last update of the dataset
> 
> It is a reflection of what the publisher writes. Not all publishers use it. And there is no guarantee that it is updated. But the EML standard says that it should be updated every time data or metadata changes. This date is not under GBIF control

This is correct @MortenHofft. It is the [pubDate](https://eml.ecoinformatics.org/schema/eml-resource_xsd.html#ResourceGroup_pubDate) field in EML which is auto published by the IPT afaik whenever it creates a new archive. It is the only date we have that tells us about the actual age of the original data - but as you say it might be missing and could even be incorrect (just as any other metadata).

I also think the current labels are very fine right now for those 3 dates. It would just indeed be good to indicate that publication date is asserted by the publishers metadata:

> Registration date March 2, 2011
> Metadata last modified April 26, 2021
> Publication date March 3, 2021

Alternatively how about "First registered with GBIF" or "GBIF registration date" for the registration date?

**Comment by timrobertson100** [comment]
Also bear in mind that `pubDate` covers the situation that a researcher publishers a paper and includes a CSV in a supplementary file. Sometime later, a meta.xml is crafted to describe the CSV adequately so that it can be read by DwC-A readers, the metadata converted into EML, and a dataset is registered in GBIF. In this scenario, the pubDate is still the date the data was published (i.e. first shared) on the internet.

The IPT doesn't support this use case, as it will set the EML pubDate to the time the dataset it generates is created, but it can be done manually (and was done back in the day).
